### PR TITLE
l10n: Delete one percent sign

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by rakekniven — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by Valdnet** [comment]
Please merge 😉.

**Comment by Valdnet** [comment]
@skjnldsv Do you have any idea why errors are occurring and how to fix them? I haven't found a way to do it.

**Comment by skjnldsv** [comment]
Ah, the double percent is here to be escaped I think.
otherwise it will be seen as a variable-substitution character :)

Is the double % causing issues on transifex?

**Review by skjnldsv — CHANGES_REQUESTED** [review]
image

**Comment by Valdnet** [comment]
> Is the double % causing issues on transifex?

@skjnldsv For the translation it does not really matter, but there may be translations where the translator will omit one `%` character thinking that it is correct. It is not the best solution. Is it not possible to use the `\ `sign?

**Comment by skjnldsv** [comment]
> Is it not possible to use the `\ `sign?

I don't think so :thinking:

**Comment by Valdnet** [comment]
And in ASCII?

**Comment by rakekniven** [comment]
> Is the double % causing issues on transifex?

No. But the translators with no developer background may have an issue with it and omit one `%` like Valdnet already wrote.

So adding a comment will solve it.
@Valdnet Would you please add a comment to the source code? Details can be found [here](https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html?highlight=comment#hints).

**Comment by Valdnet** [comment]
> Would you please add a comment to the source code? Details can be found [here](https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html?highlight=comment#hints).

Yes I can add. I thought about it too, but we will wait for the answer from @skjnldsv to see if it finds another solution.

**Comment by Valdnet** [comment]
I tried different encodings and found no other solution. I added a comment for translators and made additional corrections to another text string.

File: templates/page.values.php [hunk 1]
```diff
@@ -41,11 +41,14 @@
...
 				<span class="icon-shared"></span>
-				<h3><?php p($l->t('Open Standards and Interoperability')); ?></h3>
+				<h3><?php p($l->t('Open standards and interoperability')); ?></h3>
 			</li>
 			<li>
 				<span class="icon-user"></span>
-				<h3><?php p($l->t('100%% Open Source & community-focused')); ?></h3>
+				<h3><?php 
+					// TRANSLATORS NOTE: The characters %% should be left unchanged. This is a workaround for PHP code.
+					p($l->t('100%% open source & community-focused'));
```
Comment (skjnldsv) [inline templates/page.values.php:50]: This is not a "workaround", just the way translations works :)
We could also just use plain-text like :shrug: 

```suggestion
					p($l->t('One-hundred percent open source & community-focused'));
```

**Comment by skjnldsv** [comment]
Please also rebase and squash your commits into one for cleaner git history

File: templates/page.values.php (hunk 1 above)
Comment (Valdnet) [reply templates/page.values.php]: I shortened the comment. For translators, this information is all they need to be wary of the `%%` characters.

**Comment by Valdnet** [comment]
> Please also rebase and squash your commits into one for cleaner git history

I cannot squash because I was directly on GitHub. Please squash while merging. Thanks 😉.

**Comment by Valdnet** [comment]
\ rebase

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**